src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let localState = { ...state };
```

Comment:
Let's try to use here const instead of let

Reply:
But if I use const here, I won't be able to assign a new value to this variable...I will have to use a separate variable for the new objects 'addProperties', 'removeProperties' and 'clear'?